README.md

```diff
@@ -73,35 +74,183 @@ Array of objects representing the pyramid layout and transformation relationship
+multiscales/
+├── 0/           # Referenced by asset: "0"
+│   └── data
+└── 1/           # Referenced by asset: "1"
+    └── data
+```
+
+**Direct array layout** (COG-style):
+```json
+{
+  "layout": [
+    {"asset": "0", "scale": [1.0, 1.0]},
+    {"asset": "1", "derived_from": "0", "scale": [2.0, 2.0]}
+  ]
+}
+```
+Structure:
+```
+multiscales/
+├── 0            # Zarr Array referenced by asset: "0"
+└── 1            # Zarr Array referenced by asset: "1"
+```
+This pattern is a natural translation of COG (Cloud Optimized GeoTIFF) overviews to Zarr, where each level is stored as a separate array.
+
+**Nested array layout**:
+```json
+{
+  "layout": [
+    {"asset": "0/data", "scale": [1.0, 1.0]},
+    {"asset": "1/data", "derived_from": "0/data", "scale": [2.0, 2.0]}
+  ]
+}
+```
+Structure:
+```
+multiscales/
+├── 0/
+│   └── data     # Referenced by asset: "0/data"
+└── 1/
+    └── data     # Referenced by asset: "1/data"
+```
+
+**Nested group layout**:
+```json
+{
+  "layout": [
+    {"asset": "resolutions/full", "scale": [1.0, 1.0]},
+    {"asset": "resolutions/half", "derived_from": "resolutions/full", "scale": [2.0, 2.0]}
+  ]
+}
+```
+Structure:
+```
+multiscales/
+└── resolutions/
+    ├── full/    # Referenced by asset: "resolutions/full"
+    │   └── data
+    └── half/    # Referenced by asset: "resolutions/half"
+        └── data
+```
+
+#### Best Practices
+
+- **Consistency**: Use the same referencing style (group or array) throughout a layout
+- **Clarity**: For array-based layouts, include the full path to the array (e.g., `"0/data"`)
+- **Compatibility**: Group-based layouts are more common and may have better tool support
+
+### Transform Object
+
+The `transform` object provides a flexible mechanism to describe **relative** coordinate transformations between resolution levels. This object contains parameters that describe how to transform coordinates from a source level to the current level.
+
+#### Relative Transformations (inside `transform` object)
+
+For general-purpose transformations, the `transform` object MAY contain:
+
+|                   | Type       | Description                                                                                                                           | Required |
+| ----------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **scale**         | `number[]` | Array of scale factors per axis describing the coordinate transformation from the source level (`derived_from`) to this level        | No       |
+| **translation**   | `number[]` | Array of translation offsets per axis in the coordinate space                                                                         | No       |
+| **factors**       | `number[]` | Array of resampling factors per axis describing this level's resolution characteristics relative to a reference level                | No       |
```

Comment:
```suggestion
```
Factors was removed from everywhere else, so I'm assuming it should be here as well? Otherwise, you should probably keep the explanation about its relation to scale and an example